packages/expo-updates/src/Updates.ts

```diff
@@ -115,13 +115,23 @@ export const createdAt: Date | null = ExpoUpdates.commitTime
   ? new Date(ExpoUpdates.commitTime)
   : null;
 
-const isUsingDeveloperTool = !!(manifest as any).developer?.tool;
-const isUsingExpoDevelopmentClient = NativeModulesProxy.ExponentConstants?.appOwnership === 'expo';
-const manualUpdatesInstructions = isUsingExpoDevelopmentClient
-  ? 'To test manual updates, publish your project using `expo publish` and open the published ' +
-    'version in this development client.'
-  : 'To test manual updates, make a release build with `npm run ios --configuration Release` or ' +
-    '`npm run android --variant Release`.';
+/**
+ * In normal development, we block accessing the updates API methods on the JS side, but when developing in
+ * Expo Go or a development client build, the controllers should have control over which API methods should
+ * be allowed.
+ */
+const shouldDeferToNativeForAPIMethodAvailabilityInDevelopment =
+  !!ExpoUpdates.shouldDeferToNativeForAPIMethodAvailabilityInDevelopment;
+
+/**
+ * Developer tool is set when a project is served by `expo start`.
+ */
+const isUsingDeveloperTool =
+  'extra' in manifest ? !!manifest.extra?.expoGo?.developer?.tool : false;
+
+const manualUpdatesInstructions =
+  'To test usage of the expo-updates JS API in your app, make a release build with `npm run ios --configuration Release` or ' +
+  '`npm run android --variant Release`.';
```

Comment:
Shouldn't these commands be `npx expo run:ios --configuration Release` and `npx expo run:android --variant release`? 
https://docs.expo.dev/more/expo-cli/#compiling-android